packages/zipper/index.ts

```diff
@@ -1,5 +1,21 @@
 import { resolve } from 'path';
 import { zipBundle } from './lib/zip-bundle';
+import fs from 'fs';
+
+// Let's check if dist has correct browser build
+const distDir = resolve(__dirname, '../../dist');
+const manifestPath = resolve(distDir, 'manifest.json');
+const manifest = fs.readFileSync(manifestPath, 'utf-8');
+
+if (process.env.__FIREFOX__) {
+  if (!manifest.includes('browser_specific_settings')) {
+    throw new Error('Build is not for Firefox');
+  }
+} else {
+  if (manifest.includes('browser_specific_settings')) {
+    throw new Error('Build is not for Chrome');
+  }
+}
```

Comment:
```suggestion
  let errorMessage;
  const isFirefox = process.env.__FIREFOX__ === 'true';
  const isSpecificSettingsIncluded = manifest.includes('browser_specific_settings');
  
  if (isFirefox && !isSpecificSettingsIncluded) {
       errorMessage = "Build isn't for Firefox";
   } else if (!isFirefox && isSpecificSettingsIncluded) {
       errorMessage = "Build isn't for Chrome";
   }
   
   if (errorMessage) {
       throw new Error(errorMessage);
   }
```

I was editting it only with `suggestion` maybe this should be formatted :)

Reply:
Matter of taste probably, I find it more verbose and not clearer, anyway, thank you for suggestion 👍 

BTW, this check was caused some by some flaky turbo - zipper behavior, which shouldn't be the case, if properly done `build` then `zip` then `e2e`, but I kept it for debugging in case someone have this problem.